Marshal/Marshaling.swift

```diff
@@ -14,5 +14,5 @@
 import Foundation
 
 public protocol Marshaling {
-    func marshal() -> MarshaledObject
+    func marshal() -> Marshal
 }
```

Comment:
My usage must be much different than yours because I do quite a bit of what you don't do. Passing it into another `init(object:)` or using the `update(object:)`.